File: coding-test/백준/이진수합.py
# https://www.acmicpc.net/problem/8741

# 2024.02.21

import sys

k = int(sys.stdin.readline())
# k = 3

n = 2**k-1
# 그냥 /하면 오류남
sumsum = n*(n+1)//2

print(bin(sumsum)[2:])

File: coding-test/백준/기본/이진수합.js
// https://www.acmicpc.net/problem/8741

/**
 * @Date 2024.02.21
 */

// 왜 안되는지 모르겠음..

const fs = require('fs');
const k = Number(fs.readFileSync('/dev/stdin'));
// const k = 3;

const n = 2 ** k - 1;
const sum = (n * (n + 1)) / 2;
console.log(sum.toString(2));
